HHS/Head-Start-TTADP
# [TTAHUB-412] More flexible date picker behavior

## Description of change

On the date range selector component, you can now choose any start date after 9/1/2020. If you choose a start date after your chosen end date, the end date will adjust and the delta between them is preserved.

I did not change the filters on the Activity Report table as its a separate component and we will soon be retiring it.

## How to test
Go to the date range selector on the Dashboard or the Grantee TTA Record. Choose a start date. Choose an end date. Change your start date to be after your end date. Note that the end date is changed and the delta between them is the same.

## Issue(s)

* https://ocio-jira.acf.hhs.gov/browse/TTAHUB-412


## Checklists

### Every PR


- [n/a] API Documentation updated
- [n/a] Boundary diagram updated
- [n/a] Logical Data Model updated
- [n/a] [Architectural Decision Records](https://adr.github.io/) written for major infrastructure decisions

### Production Deploy


### After merge/deploy
